style.css

```diff
@@ -1,2 +1,199 @@
-body { font-family: "Montserrat", serif; }
+body { 
```

Comment:
Codacy found an issue: [Expected newline after "{" of a multi-line block (block-opening-brace-newline-after)](https://app.codacy.com/gh/WebHeroSchool/IFW/pullRequest?prid=6044074)